corehq/motech/repeaters/expression/repeaters.py

```diff
@@ -63,6 +88,40 @@ def get_url(self, repeat_record):
             )
         return base_url
 
+    def handle_response(self, response, repeat_record):
+        super().handle_response(response, repeat_record)
+        if self.case_action_filter_expression and is_response(response):
+            try:
+                self._process_response_as_case_update(response, repeat_record)
+            except Exception as e:
+                notify_exception(None, "Error processing response from Repeater request", e)
+
+    def _process_response_as_case_update(self, response, repeat_record):
+        domain = repeat_record.domain
+        context = get_evaluation_context(domain, repeat_record, self.payload_doc(repeat_record), response)
+        if not self.parsed_case_action_filter(context.root_doc, context):
+            return False
+
+        self._perform_case_update(domain, context)
+        return True
+
+    def _perform_case_update(self, domain, context):
+        data = self.parsed_case_action_expression(context.root_doc, context)
+        if data:
+            data = data if isinstance(data, list) else [data]
+            handle_case_update(
+                domain=domain,
+                data=data,
+                user=UserDuck('system', ''),
+                device_id=self.device_id,
+                is_creation=False,
+                xmlns=REPEATER_RESPONSE_XMLNS,
```

Comment:
nit: do we need the XMLNS to be more specific to case expression repeater response?

Reply:
nope, can also be a form repeater